README.md

```diff
@@ -22,12 +22,12 @@ Alternatively, to install the latest binary release:
 
 ```
 CILIUM_CLI_VERSION=$(curl -s https://raw.githubusercontent.com/cilium/cilium-cli/main/stable.txt)
-GOOS=$(go env GOOS)
-GOARCH=$(go env GOARCH)
-curl -L --remote-name-all https://github.com/cilium/cilium-cli/releases/download/${CILIUM_CLI_VERSION}/cilium-${GOOS}-${GOARCH}.tar.gz{,.sha256sum}
-sha256sum --check cilium-${GOOS}-${GOARCH}.tar.gz.sha256sum
-sudo tar -C /usr/local/bin -xzvf cilium-${GOOS}-${GOARCH}.tar.gz
-rm cilium-${GOOS}-${GOARCH}.tar.gz{,.sha256sum}
+CLI_ARCH=amd64
+if [ "$(uname -m)" = "aarch64" ]; then CLI_ARCH=arm64; fi
+curl -L --fail --remote-name-all https://github.com/cilium/cilium-cli/releases/download/${CILIUM_CLI_VERSION}/cilium-linux-${CLI_ARCH}.tar.gz{,.sha256sum}
+sha256sum --check cilium-linux-${CLI_ARCH}.tar.gz.sha256sum
+sudo tar xzvfC cilium-linux-${CLI_ARCH}.tar.gz /usr/local/bin
+rm cilium-linux-${CLI_ARCH}.tar.gz{,.sha256sum}
```

Comment:
This now hard-codes `linux`. Maybe it's better to replace this section entirely with a link to the docs which have installation instructions for all supported operating systems?